### DISCUSSION TITLE
Welcome to alexa_media_player Discussions!

### DISCUSSION THREAD

**Discussion by alandtse** [alandtse/alexa_media_player#1036] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pittgo** [comment]
I am having an intersting problem.   have Alexa media player installed on two systems with two separate Amazon accounts and two separate Nabu Casa cloud accounts. When i register one machine with Amazon,,, it  degegisters the other machine.   What is noteworthy is that both machines at one time ran the same HA account amazon and Nabu Casa however was reconfigured when I replaced that machine with an RPI and then used the original to replace an older machine . They are not  at the same address nor on a shared network in anyway.  The Amazon account are not linked.  No errors occur  the only thing that happens is as we reauthorize the one the other location deauthorizes.    

I have removed and resinstalled HACS and Alexa Media Player on both machines thinking the accounts are linked some how but same problem

Is there a file i should be deleting specifically that is holding some of the previous Amazon linking detail ?

Any help wpould be appreciated  

G
